components/lsp-form/Collateral.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from "react";
+import { FormRow, BalanceRow, iconStyles } from "./LSPForm.styled";
+import EthIcon from "../../public/icons/eth-icon.svg";
+import UniswapIcon from "../../public/icons/uniswap-logo.svg";
+import Dropdown from "../dropdown";
+import TextInput from "../text-input";
+import { DropdownVariant } from "../dropdown/Dropdown";
+import { LabelPlacement } from "../text-input/TextInput";
+import useWindowSize from "../../hooks/useWindowSize";
+
+interface Props {
+  collateral: string;
+  setCollateral: React.Dispatch<React.SetStateAction<string>>;
+  amount: string;
+  setAmount: React.Dispatch<React.SetStateAction<string>>;
+  // Adjust CSS slightly if its the redeem form or the mint form.
+  redeemForm?: boolean;
+  collateralOnTop?: boolean;
+}
+
+const Collateral: FC<Props> = ({
+  collateral,
+  setCollateral,
+  amount,
+  setAmount,
+  collateralOnTop,
+  redeemForm,
+}) => {
+  const size = useWindowSize();
+  const width = size.width && size.width > 728 ? "230px" : "100%";
+  return (
+    <>
+      <FormRow>
+        {collateral === "eth" && <EthIcon style={iconStyles} />}
+        {collateral === "uniswap" && <UniswapIcon style={iconStyles} />}
+        <Dropdown
+          setValue={setCollateral}
+          variant={"coin" as DropdownVariant}
+          defaultValue={{ label: "ETH", value: "eth" }}
+          items={[
+            {
+              label: "ETH",
+              value: "eth",
+            },
+            { label: "UNI", value: "uniswap" },
```

Comment:
i assume this will eventually be props or something?

Reply:
Yeah, 100%. I assume this will have to come from the API or something, as I'm not even sure what supporting elements we have. Actually, given the collateral might be something we don't normally support, we might not even be able to show the image.